validator/modules/onnx/train_svm.py

```
@@ -0,0 +1,160 @@
+from skl2onnx import to_onnx
+from skl2onnx.common.data_types import FloatTensorType
+
+
+def load_and_preprocess_data(csv_path):
+    """Load and preprocess the training data"""
+    print(f"Loading data from {csv_path}")
+
+    df = pd.read_csv(csv_path)
+    print(f"Data shape: {df.shape}")
+    print(f"Columns: {df.columns.tolist()}")
+    print(f"First few rows:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)